sdcm/utils/docker_remote.py

```diff
@@ -61,7 +61,7 @@ def private_ip_address(self):
 
     @property
     def cql_address(self):
-        return self.internal_ip_address
+        return self.internal_ip_address  # '127.0.0.1' if self.running_in_podman(self.node) else
```

Comment:
This one looks like a debug-leftover.

Reply:
it's a left over from when I was tried to overcome the issue of network with podman, which I gave up on solving in this PR

Reply:
I'll remove it

Reply:
removed